baselines/asset/samples/generated/v1/asset_service.batch_get_assets_history.js.baseline

```diff
@@ -0,0 +1,80 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+'use strict';
+
+function main() {
+  // [START asset_batch_get_assets_history_sample]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  /**
+   *  Required. The relative name of the root asset. It can only be an
+   *  organization number (such as "organizations/123"), a project ID (such as
+   *  "projects/my-project-id")", or a project number (such as "projects/12345").
+   */
+  // const parent = 'abc123'
+  /**
+   *  A list of the full names of the assets. For example:
+   *  `//compute.googleapis.com/projects/my_project_123/zones/zone1/instances/instance1`.
+   *  See [Resource
```

Comment:
Nit: part of me wishes we removed markdown links, but I think this is a "nice to have" we could do in follow up work.

Reply:
Let's track this in a feature request so we don't lose it?